kevintdd/src/main/java/TaxCalculator.java

```diff
@@ -0,0 +1,43 @@
+import java.math.BigDecimal;
```

Comment:
Look at proposed changes in attachment, tests could have to be adjusted to account for corrected totals
[TaxCalculator.txt](https://github.com/KTBG7/tdd/files/4956423/TaxCalculator.txt)